grails-app/controllers/org/pih/warehouse/putaway/PutawayController.groovy

```diff
@@ -46,9 +46,7 @@ class PutawayController {
             return
         }
 
-        List<PutawayTask> tasks = order.orderItems.collect { orderItem ->
-            PutawayTaskAdapter.toPutawayTask(orderItem)
-        }.findAll { it != null }
+        List<PutawayTask> tasks = putawayTaskService.search(order.destination, null, null, null, order, params)
```

Comment:
yeah this what i mean. it's probably better to have a method for facility/product and another for facility/order.